pkg/server.go

```diff
@@ -59,7 +85,6 @@ func Start(cfg *Config) {
 	logrusLevel, err := logrus.ParseLevel(cfg.LogLevel)
 	if err != nil {
 		logrus.WithError(err).Fatal("unable to set the log level")
-		logrusLevel = logrus.ErrorLevel
```

Comment:
AFAIU this line can't be reached if err != nil because the previous statement will panic.

Reply:
cc @PeterYurkovich

Reply:
Thanks for spotting that. I threw up #188 to fix it.